src/modules/users/sagas/user.ts

```diff
@@ -209,10 +219,157 @@ function* userLogout() {
   return null;
 }
 
+function* userDepositToken({
+  meta,
+  payload: { tokenAddress, amount, colonyAddress },
+}: Action<ActionTypes.USER_DEPOSIT_TOKEN>) {
+  const txChannel = yield call(getTxChannel, meta.id);
+  try {
+    const apolloClient = TEMP_getContext(ContextModule.ApolloClient);
+    const colonyManager = TEMP_getContext(ContextModule.ColonyManager);
+    const { walletAddress } = yield getLoggedInUser();
+
+    // eslint-disable-next-line max-len
```

Comment:
Please leave a note when it's not obvious why you disabled the `max-len` rule.

_(I'm assuming, in this case, it's because `prettier` doesn't suggest a good solution)_

Reply:
That's correct, I'll leave a comment then